src/index.html

```diff
@@ -4,9 +4,294 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="./index.html" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="header__list">
+          <li>
+            <a class="header__list-link is-active" href="#Apple">
+              Apple
+            </a>
+          </li>
+          <li>
+            <a class="header__list-link" href="#Samsung">
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a class="header__list-link" href="#Smartphones">
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a class="header__list-link" href="#Laptops"
+            data-qa="nav-hover">
+            Laptops & Computers
+            </a>
```

Comment:
Need to check and fix it everywhere
```suggestion
            <a
              class="header__list-link"
              href="#Laptops"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```